### PR TITLE
app-text/cherrytree: version bump to 0.38.4

### PR DESCRIPTION
**Pull request by Anth0rx**
Version 0.37.0 is from 05/2016. Version 0.38.4 is the most recent one from 12/2017.

### CODE REVIEW

File: app-text/cherrytree/cherrytree-0.38.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pyenchant[${PYTHON_USEDEP}]
+	>=dev-python/pygtk-2.16:2[${PYTHON_USEDEP}]
+	dev-python/pygtksourceview:2[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )
+"
+
+PLOCALES='cs de es fr hy it ja lt nl pl pt_BR ru sl tr uk zh_CN'
```
Comment (Zlogene) [inline app-text/cherrytree/cherrytree-0.38.4.ebuild]: why is this in the middle? move this straight after `EAPI` defenition

File: app-text/cherrytree/cherrytree-0.38.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+inherit l10n
```
Comment (Zlogene) [inline app-text/cherrytree/cherrytree-0.38.4.ebuild]: the same, I have no clue why you defined `inherit` twice at all

File: app-text/cherrytree/cherrytree-0.38.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+DISTUTILS_SINGLE_IMPL=true
+
+inherit xdg distutils-r1
+
+DESCRIPTION='A hierarchical note taking application'
+HOMEPAGE='https://www.giuspen.com/cherrytree'
+LICENSE='GPL-3'
+
+SLOT='0'
+SRC_URI="https://github.com/giuspen/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (Zlogene) [inline app-text/cherrytree/cherrytree-0.38.4.ebuild]: should go straight after the `HOMEPAGE` and before the `LICENSE`

File: app-text/cherrytree/cherrytree-0.38.4.ebuild (hunk 1 above)
Comment (Zlogene) [inline app-text/cherrytree/cherrytree-0.38.4.ebuild]: also it is good practice to declare the `BACKUP_LOCALE` variable

**Comment by Anth0rx** [comment]
I simply did a version bump. I did not change the ebuild at all. But I will change the requested parts,

File: app-text/cherrytree/cherrytree-0.38.4.ebuild [hunk 4]
```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
```
Comment (mgorny) [inline app-text/cherrytree/cherrytree-0.38.4.ebuild]: I'm sorry but this is obsolete now. Could you update it for the new version? Please also do the complete background check, including checking for dependency and build system changes.

File: app-text/cherrytree/cherrytree-0.38.4.ebuild (hunk 1 above)
Comment (mgorny) [reply app-text/cherrytree/cherrytree-0.38.4.ebuild]: Actually, please remove the whole l10n+nls logic. We don't do that stuff for Python apps.

**Comment by mgorny** [comment]
I'm sorry but I'm going to close this for inactivity. Feel free to reopen if you are around once again and are willing to update it. Also please note that you need to reopen it before pushing new commits, as otherwise GitHub will block reopening.

File: app-text/cherrytree/cherrytree-0.38.4.ebuild (hunk 4 above)
Comment (Anth0rx) [reply app-text/cherrytree/cherrytree-0.38.4.ebuild]: What exactly is obsolete? Can you tell me how to do a complete background check including checking for dependency and build system changes?

**Pull request closed without merging**